package/MDAnalysis/core/universe.py

```diff
@@ -901,7 +941,8 @@ def _process_attr(self, attr):
                   'residue': self._topology.n_residues,
                   'segment': self._topology.n_segments}
         logger.debug("_process_attr: Adding {0} to topology".format(attr))
-        if (attr.per_object is not None and len(attr) != n_dict[attr.per_object]):
+        if (attr.per_object is not None and len(
+                attr) != n_dict[attr.per_object]):
```

Comment:
I've seen a few lines re-formatted a bit weirdly like this now -- are you using an autoformatter? If so, could you please try to avoid using it -- it can lead to spurious "changes" like this one here (where I don't think the code has actually changed), and also the breaking of the `len()` function is quite odd.